README.md

```diff
@@ -1,2 +1,9 @@
 # chemnlp
-ChemNLP project
+
+ChemNLP is a project that aims to create an extensive chemistry dataset and use it to train large language models (LLMs) for a wide range of chemistry applications. The resulting models will be capable of predicting properties of materials and molecules, chemical reactions, and even generating computational chemistry code. The potential use cases include accelerating drug discovery, reducing the need for animal experiments, and bringing machine learning to more complex tasks in chemistry. The project builds on prior work that has shown basic chemistry capabilities in LLMs and will use a model setup similar to Med-PaLM and "instruction prompt tuning" (IPT) to fine-tune the LLMs with a few trainable parameters.
+
+## Relevant Links
+**Project Proposal**: https://docs.google.com/document/d/1C44EKSJRojm39P2CaxnEq-0FGwDRaknKxJ8lZI6xr5M/edit?usp=sharing
+**Chemistry datasets repo**: https://github.com/kjappelbaum/awesome-chemistry-datasets
+**Github task board**: https://github.com/orgs/OpenBioML/projects/5/views/1
```

Comment:
I just added a dataset board too: https://github.com/orgs/OpenBioML/projects/6/views/1

Reply:
we don't use that anymore